app/views/product_users/index.html.haml

```diff
@@ -19,7 +19,7 @@
 - elsif @product_users.empty?
   %p.notice= text("none")
 - else
-  - if @product.respond_to?(:product_access_groups) && @product.product_access_groups.any?
+  - if @product.has_access_list?
```

Comment:
I feel like this name is misleading. This isn't checking for an access list (that's `requires_approval?`). it's checking for product access groups (which are related to schedule rules, so only instruments and rooms have them). I like the DRYing here, but maybe we should rename it? `has_product_access_groups?` perhaps?